src/app/mail/mail-sidebar/mail-sidebar.component.html

```diff
@@ -43,19 +43,19 @@
                         </ng-container>
                     </a>
                 </li>
-                <li class="mailbox-sidebar-nav-item"  routerLinkActive="active">
-                    <a routerLink="/mail/draft/page/1" class="mailbox-sidebar-nav-link" (click)="toggleMenu()">
+                <li class="mailbox-sidebar-nav-item"  [class.active]="mailState?.currentFolder === mailFolderType.DRAFT">
+                    <a routerLink="/mail/draft/page/1" class="mailbox-sidebar-nav-link" (click)="toggleMenu('draft')">
```

Comment:
why did you pass `draft` in toggleMenu function

Reply:
after changing issue resolving strategy, forgot to remove